### PR TITLE
feat(service): add postcard service + test

### PR DESCRIPTION
**Pull request by lvamaral**
what: add postcard service and associated test
why: so that we can make api calls to create postcards in our app by injecting this service

### CODE REVIEW

**Review by thetonywu — APPROVED** [review]

File: app/services/postcard.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var Config = require('../../config');
```
Comment (jesscxu) [inline app/services/postcard.js:3]: nit: use `const` instead here and throughout file

File: app/services/postcard.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+module.exports = function (API) {
```
Comment (jesscxu) [inline app/services/postcard.js:5]: nit: arrow function

File: app/services/postcard.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+  var Postcard = {};
+
+  Postcard.url = Config.API_HOST + '/postcards';
```
Comment (jesscxu) [inline app/services/postcard.js:9]: nit: use es6 string interpolation `${Config.API_HOST}/postcards`

File: app/services/postcard.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+  Postcard.create = function (payload) {
```
Comment (jesscxu) [inline app/services/postcard.js:11]: nit: arrow function

File: test/services/postcard.spec.js [hunk 5]
```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+var expect  = require('chai').expect;
```
Comment (jesscxu) [inline test/services/postcard.spec.js]: `const`

File: test/services/postcard.spec.js [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+var Angular = require('angular');
```
Comment (jesscxu) [inline test/services/postcard.spec.js]: nit: alphabetize

File: test/services/postcard.spec.js [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+var Config = require('../../config');
+
+var $q;
+var $rootScope;
+var API;
+var Postcard;
+
+require('angular-mocks');
+
+describe('postcard service', function () {
```
Comment (jesscxu) [inline test/services/postcard.spec.js:16]: nit: arrow function here and throughout rest of file

**Review by jesscxu — CHANGES_REQUESTED** [review]
looks good so far! i think miyagi is not up to date since we started using es6 javascript conventions...so in general if a file is new like this for example, use these conventions, but if you are editing an old file, just use old conventions to keep the file consistent

**Comment by lvamaral** [comment]
@jesscxu  since the entire codebase for this forked project is in the old syntax I thought it would be better to have the new files be consistent with the syntax, per a conversation I had w/ tony. I've alphabetized the declarations; can I merge it as it currently is?

**Comment by jesscxu** [comment]
yeah since you forked it, i think you can leave it for consistency...forgot this was a forked repo!! go for it!

**Review by jesscxu — APPROVED** [review]
LGTM

**Pull request merged**